### PR TITLE
[MTM-56608] Fixed issue with logging using user alias and TFA SMS wit…

### PR DESCRIPTION
**Pull request by Michal-Bak**
Release notes which go only to:
10.18.540.63
10.16.0.462
10.17.0.483
10.18.0.373

(without develop)

Fixed issue with logging using user alias and TFA SMS with basic authentication

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10-18-540-63-fixed-issue-with-logging-using-user-alias-and-tfa-sms-with-basic-authentication.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-56608
+version: 10.18.540.63
```
Comment (kalavathig1) [inline content/change-logs/platform-services/cumulocity-10-18-540-63-fixed-issue-with-logging-using-user-alias-and-tfa-sms-with-basic-authentication.md:15]: you have fixed it in multiple release lines, should you create release notes for each of the fixed version ?

Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10-18-540-63-fixed-issue-with-logging-using-user-alias-and-tfa-sms-with-basic-authentication.md:15]: @kalavathig1 @Michal-Bak For the release lines 10.18 and below I will add the ticket to the release notes.

Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10-18-540-63-fixed-issue-with-logging-using-user-alias-and-tfa-sms-with-basic-authentication.md:15]: > @kalavathig1 @Michal-Bak For the release lines 10.18 and below I will add the ticket to the release notes. Please still add the text to the release notes field in such cases (older release lines).

Comment (Michal-Bak) [reply content/change-logs/platform-services/cumulocity-10-18-540-63-fixed-issue-with-logging-using-user-alias-and-tfa-sms-with-basic-authentication.md:15]: Yes, after this base PR is accepted I will prepare graft as always.

File: content/change-logs/platform-services/cumulocity-10-18-540-63-fixed-issue-with-logging-using-user-alias-and-tfa-sms-with-basic-authentication.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+Fixed a random *"invalid TFA token due to user inactivity"* issue when logging in using user alias and TFA SMS with basic authentication.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-18-540-63-fixed-issue-with-logging-using-user-alias-and-tfa-sms-with-basic-authentication.md:17]:
```suggestion
Fixed a random issue ("invalid TFA token due to user inactivity") when logging in using the user alias and TFA SMS with basic authentication.
```

File: content/change-logs/platform-services/cumulocity-10-18-540-63-fixed-issue-with-logging-using-user-alias-and-tfa-sms-with-basic-authentication.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed issue with logging using user alias and TFA SMS with basic authentication
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-18-540-63-fixed-issue-with-logging-using-user-alias-and-tfa-sms-with-basic-authentication.md:3]:
```suggestion
title: Login using user alias and TFA SMS with basic authentication works reliably again
```

**Review by kalavathig1 — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**